packages/table/test/selectionTests.tsx

```diff
@@ -96,6 +97,26 @@ describe("Selection", () => {
         expect(onSelection.lastCall.args).to.deep.equal([[]], "meta key clear");
     });
 
+    it("Transforms regions on selections", () => {
+        const selectedRegionTransform = () => {
+            return Regions.row(1);
+        };
+        const onSelection = sinon.spy();
+        const table = harness.mount(createTableOfSize(3, 7, {}, {onSelection, selectedRegionTransform}));
+
+        // clicking adds transformed selection
+        table.find(CELL_SELECTOR).mouse("mousedown").mouse("mouseup");
+
+        expect(onSelection.called).to.equal(true);
```

Comment:
nit: I tend to prefer `expect(onSelection.called).to.be.true` syntax for boolean checks, because it reads a little more like plain English.